audit/projects/kubernetes-public/services/container/clusters.txt

```diff
@@ -1,2 +1,2 @@
-aaa	us-central1	us-central1-c;us-central1-a;us-central1-f	8	RUNNING
+aaa	us-central1	us-central1-c;us-central1-a;us-central1-f	6	RUNNING
```

Comment:
looks like we shrank the node count for aaa? but haven't adjusted terraform files at all?

Reply:
It's just a result of autoscalling I think. We didn't change anything particular during the meeting. Just one service account was created for me playing with some automation testing. I'm gonna give you some quick update about what we were discussing at a meeting, but please give me few hours to finish other stuff today.